nodewatcher/modules/monitor/http/survey/management/commands/export_survey_data.py

```diff
@@ -0,0 +1,137 @@
+import io
+import json
+import datetime
+
+from django.core.management import base
+
+from django_datastream import datastream
+
+
+class Command(base.BaseCommand):
+    help = "Consolidates the latest survey data within the specified time interval \
+           from all nodes and exports the meta graph as a JSON file."
+
+    def add_arguments(self, parser):
+
+        # Optional arguments
+        parser.add_argument(
+            '--datetime',
+            type=str,
+            action='store',
+            dest='datetime',
+            help="All the data will be collected between this UTC datetime \
+                 (format 'yyyy-mm-ddThh:mm:ss' (i.e. '2007-03-04T12:00:00 2007-04-10T12:00:00'), \
```

Comment:
It does not matter in which way they are stored. The whole point of using timezone-aware timestamps is that Python knows how to convert between them.

But this should not matter at all if you are using Django date parsing anyway.